src/components/InventoryGroups/Modals/AddSystemsToGroupModal.js

```diff
@@ -163,6 +160,19 @@ const AddSystemsToGroupModal = ({
   const ConventionalInventoryTable = (
     <InventoryTable
       columns={(columns) => prepareColumns(columns, false, true)}
+      getEntities={async (items, config, showTags, defaultGetEntities) =>
+        await defaultGetEntities(
+          items,
+          {
+            ...config,
+            filters: {
+              ...config.filters,
+              hostTypeFilter: 'nil', // request only conventional systems
```

Comment:
We could have a `defaultConventionalSystemsGetEntities` that sets the filter. That way it would already explain in the function name what it is/does and it wouldn't need a comment to further explain.